app/facades/food_facade.rb

```diff
@@ -0,0 +1,10 @@
+class FoodFacade
+  def self.foods(food)
+    json = FoodService.search_foods(food)
+    count = json[:totalHits]
+    json[:foods].map do |food_data|
+      Food.new(food_data, count)
```

Comment:
Neat choice to include the count here! A counter argument to this approach might be that you are storing count on every `Food` item which ultimately takes up space in memory. It's not a huge lift and I don't mind, but for a large scale app or if you were handling a lot of data, it could be something that would come up.